packages/bpk-component-text/src/BpkText.scss

```diff
@@ -0,0 +1,27 @@
+@import '~bpk-mixins';
+
+.bpk-text {
```

Comment:
Yeah I know, feels kinda iffy to have `bpk-text` as a class just for the sake of it. Would `bpk-text-sm`, `bpk-text-base` be better or worse?